src/core/network.js

```diff
@@ -115,20 +133,17 @@ var NetworkManager = (function NetworkManagerClosure() {
         pendingRequest.expectedStatus = 200;
       }
 
-      if (args.onProgressiveData) {
-        // Some legacy browsers might throw an exception.
-        try {
-          xhr.responseType = 'moz-chunked-arraybuffer';
-        } catch(e) {}
-        if (xhr.responseType === 'moz-chunked-arraybuffer') {
-          pendingRequest.onProgressiveData = args.onProgressiveData;
-          pendingRequest.mozChunked = true;
-        } else {
-          xhr.responseType = 'arraybuffer';
-        }
+//#if (GENERIC || FIREFOX || MOZCENTRAL)
+      if (args.onProgressiveData && supportsMozChunked) {
+        xhr.responseType = 'moz-chunked-arraybuffer';
+        pendingRequest.onProgressiveData = args.onProgressiveData;
+        pendingRequest.mozChunked = true;
       } else {
         xhr.responseType = 'arraybuffer';
       }
+//#else
```

Comment:
Since `supportsMozChunked` is always set for non-Chromium builds, we can do either of the following (the FIREFOX|MOZCENTRAL build does not really need the boolean, but in my [previous PR a reviewer suggested](https://github.com/mozilla/pdf.js/pull/5531#discussion-diff-31866452) to not abuse the preprocessor to eliminate the boolean, so I've introduced the boolean). WDYT?

```
//#if CHROMIUM
//    xhr.responseType = 'arraybuffer';
//#else
      if (args.onProgressiveData && supportsMozChunked) {
        xhr.responseType = 'moz-chunked-arraybuffer';
        pendingRequest.onProgressiveData = args.onProgressiveData;
        pendingRequest.mozChunked = true;
      } else {
        xhr.responseType = 'arraybuffer';
      }   
//#endif
```

or

```
      xhr.responseType = 'arraybuffer';
//#if !CHROMIUM
      if (args.onProgressiveData && supportsMozChunked) {
        xhr.responseType = 'moz-chunked-arraybuffer';
        pendingRequest.onProgressiveData = args.onProgressiveData;
        pendingRequest.mozChunked = true;
      }   
//#endif
```

or something like https://github.com/mozilla/pdf.js/pull/5531#discussion-diff-31866452